src/__tests__/acceptance/user-controller.acceptance.ts

```diff
@@ -138,7 +138,7 @@ describe('UserController', () => {
       expect(res.body.error.message).to.equal('Invalid email address');
     });
 
-    it('password reset returns an error when invalid password is used', async () => {
+    it('(password reset) sloud return an error when invalid password is used', async () => {
```

Comment:
it('...should*...)

Reply:
Nope ca devrait etre : `it('Should.....')` ca commence par should pske c'est la suite du mot `it`

Reply:
`It password reset should returns an error` c'est pas ouf